akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -879,7 +882,7 @@ object Logging {
     // format: ON
 
     def timestamp(event: LogEvent): String = {
-      formatter.format(Instant.ofEpochMilli(event.timestamp))
+      formatter.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(event.timestamp), timeZone))
```

Comment:
@guidomedina The bug was that `Instant.ofEpochMilli` can't be formatted with that pattern. It must be a `LocalDateTime`.